.github/actions/jlab-a11y-test/action.yml

```diff
@@ -0,0 +1,116 @@
+name: Run accessibility regression tests on JupyterLab
+description: |
+  Builds JupyterLab from source, then runs accessibility regression tests
+  against it using Playwright. Outputs the results to a markdown file which is
+  then copied to the GitHub workflow run summary.
+inputs:
+  repository:
+    description: JupyterLab repository, for example gabalafou/jupyterlab
+  ref:
+    description: ref, branch, or SHA
+runs:
+  using: composite
+  steps:
+    - name: List environment variables
+      run: env
+      shell: bash -l {0}
+
+    - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+      shell: bash
+    - run: echo "🐧 This job is now running on a ${{ runner.os }} server hosted by GitHub!"
+      shell: bash
+    - run: echo "🔎 The name of your branch is ${{ github.ref }} and your repository is ${{ github.repository }}."
+      shell: bash
+    - run: |
+        echo "github.workspace = ${{ github.workspace }}"
+        echo "github.sha = ${{ github.sha }}"
+        echo "github.repository = ${{ github.repository }}"
+        echo "github.server_url = ${{ github.server_url }}"
+      shell: bash
+
+    - name: Cache conda 🧠
+      uses: actions/cache@v2
+      env:
+        # Increase this value to reset cache if environment.yml has not changed
+        CACHE_NUMBER: 0
+      with:
+        path: ~/conda_pkgs_dir
+        key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{
+          hashFiles('environment.yml') }}
+
+    - name: Install miniconda 🐍
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        miniforge-variant: Mambaforge
+        miniforge-version: latest
+        use-mamba: true
+        environment-file: ./testing/jupyterlab/environment.yml
+        activate-environment: a11y-tests
+        use-only-tar-bz2: true
+
+    - name: Sanity checks on environment 🔍
+      run: |
+        conda info
+        conda env list
+        echo $CONDA_PREFIX
+      shell: bash -l {0}
+
+    - name: Checkout JupyterLab
+      uses: actions/checkout@v3
+      with:
+        repository: ${{ inputs.repository || 'jupyterlab/jupyterlab' }}
+        ref: ${{ inputs.ref || 'master' }}
+        path: jupyterlab
+
+    - name: Build JupyterLab
+      run: |
+        conda run --prefix $CONDA_PREFIX ./scripts/ci_install.sh
+        conda run --prefix $CONDA_PREFIX jlpm run build
+      shell: bash -l {0}
+      working-directory: jupyterlab
+
+    - name: Check JupyterLab installation
+      run: |
+        which jupyter
+        jupyter lab --version
+      shell: bash -l {0}
```

Comment:
For this step, I think so? Otherwise I'm not sure the shell will be activated to the right conda environment. I got this from your code. There's a comment in the docs for the conda-incubator/setup-miniconda action about this.

I'd have to remove this line though, and run the workflow to be sure. 

Honestly, shell scripting is not my forte 😅.